sympy/plotting/plot.py

```diff
@@ -1520,6 +1522,20 @@ def _process_series(self, series, ax, parent):
                 ax.add_patch(rect)
         if parent.fill:
             ax.fill_between(**parent.fill)
+        if parent.special_points:
+            xticks = ax.get_xticks()
+            yticks = ax.get_yticks()
+            if isinstance(ax, Axes3D): zticks = ax.get_zticks()
+            for point in parent.special_points:
+                xticks = np.append(xticks, point[0])
+                yticks = np.append(yticks, point[1])
+                if isinstance(ax, Axes3D):
+                    zticks = np.append(yticks, point[2])
+                    self.plt.plot(point[0], point[1], point[2], marker="o", markersize=5, markeredgecolor="blue", markerfacecolor="blue", zorder=10)
+                else: self.plt.plot(point[0], point[1], marker="o", markersize=5, markeredgecolor="blue", markerfacecolor="blue", zorder=10)
```

Comment:
I think that the color is forced to be blue. 
Can there be some issues if this doesn't look good for other stylings or such.

Reply:
Would it be better then to allow the user to set the color, or have it match the `line_color` that's set in `plot`, or something else?